repos/Directory.Build.targets

```diff
@@ -139,6 +140,45 @@
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json"
+                         NewFeed="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/vstest/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/mstestv2/auth/1e768268-8c95-4e7e-9fd2-0eb1b1b69b18/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/roslyn/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/symreader-converter/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/interactive-window/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/mstestv2/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/vsunittesting/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/msbuild/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+      <LegacyFeedMapping Include="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json"
+                         NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
```

Comment:
It's very hard for me to make out the pairs, can you switch to indentation?

```xml
    <ItemGroup>
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/nuget-build/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://www.myget.org/F/nugetbuild/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json"
        NewFeed="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/vstest/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/mstestv2/auth/1e768268-8c95-4e7e-9fd2-0eb1b1b69b18/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/roslyn/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/symreader-converter/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/interactive-window/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/mstestv2/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/vsunittesting/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/msbuild/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
      <LegacyFeedMapping
        Include="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json"
        NewFeed="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
    </ItemGroup>
```

(I prefer indents over alignment everywhere because it means automated renames and other shifts won't wreck readability, but in this case the gap simply makes it easier to pick out which URLs are which.)